spec.html

```diff
@@ -42226,6 +42405,120 @@ <h1>Set.prototype.has ( _value_ )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-set.prototype.intersection">
+        <h1>Set.prototype.intersection ( _other_ )</h1>
+        <p>This method performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[SetData]]).
+          1. Let _otherRec_ be ? GetSetRecord(_other_).
+          1. Let _resultSetData_ be a new empty List.
+          1. If SetDataSize(_O_.[[SetData]]) ≤ _otherRec_.[[Size]], then
+            1. Let _thisSize_ be the number of elements in _O_.[[SetData]].
+            1. Let _index_ be 0.
+            1. Repeat, while _index_ &lt; _thisSize_,
+              1. Let _e_ be _O_.[[SetData]][_index_].
+              1. Set _index_ to _index_ + 1.
+              1. If _e_ is not ~empty~, then
+                1. Let _inOther_ be ToBoolean(? Call(_otherRec_.[[Has]], _otherRec_.[[Set]], « _e_ »)).
+                1. If _inOther_ is *true*, then
+                  1. NOTE: It is possible for earlier calls to _otherRec_.[[Has]] to remove and re-add an element of _O_.[[SetData]], which can cause the same element to be visited twice during this iteration.
+                  1. Let _alreadyInResult_ be SetDataHas(_resultSetData_, _e_).
+                  1. If _alreadyInResult_ is *false*, then
```

Comment:
Given we have the note above the step, the alias name isn't adding anything, and I think we can inline this.

```suggestion
                  1. If SetDataHas(_resultSetData_, _e_) is *false*, then
```

Same below.